src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/IKafkaEndpointsConfigurationBuilder.cs

```diff
@@ -38,12 +38,16 @@ public interface IKafkaEndpointsConfigurationBuilder : IEndpointsConfigurationBu
         ///     Specifies whether the producers must be immediately instantiated and connected. When <c>false</c> the
         ///     <see cref="IProducer" /> will be created only when the first message is about to be produced.
         /// </param>
+        /// <param name="idempotentRegistration">
+        ///     Only adds the endpoint when it is not already registered.
+        /// </param>
         /// <returns>
         ///     The <see cref="IKafkaEndpointsConfigurationBuilder" /> so that additional calls can be chained.
         /// </returns>
         public IKafkaEndpointsConfigurationBuilder AddOutbound<TMessage>(
             Action<IKafkaProducerEndpointBuilder> endpointBuilderAction,
-            bool preloadProducers = true);
+            bool preloadProducers = true,
```

Comment:
In this case, when using the `IKafkaProducerEndpointBuilder`, I'd expect the API To be something more fluent.

This is not really readable:
`.AddOutbound<IIntegrationEvent>(endpoint => endpoint.ProduceTo(DefaultTopicName), true, true)`

As a developer I'd prefer something like
`.AddOutbound<IIntegrationEvent>(endpoint => endpoint.ProduceTo(DefaultTopicName).AsIdempotent())

Or, maybe, we could even move this setting up one level.
```csharp
 .AddKafkaEndpoints(endpoints => endpoints
    .Configure(config => { config.BootstrapServers = "PLAINTEXT://tests"; })
    .EnableIdempotentOutboundEndpointsRegistration() //<-- ...kinda...
    .AddOutbound<IIntegrationEvent>(endpoint => endpoint.ProduceTo(DefaultTopicName), true, true)
    .AddOutbound<IIntegrationEvent>(endpoint => endpoint.ProduceTo(DefaultTopicName), true, true))
```
Besides, I think that idempotent should be the default....and I actually wonder if the parameter is of any usage. Why would one want otherwise?